story/ep1/pt/umi1_2.txt

```diff
@@ -255,17 +255,17 @@
 `"......Uu!`
 ` Vir dar água pra ela!"`
 `"Acho que você deveria dar um nome para a rosa.`
-` Tenho certeza que isso vai fazer ela feliz, e você será capaz de entendê‐la um pouco melhor também."`
+` Tenho certeza que isso vai fazer ela feliz, e você conseguirá entender ela um pouco melhor também."`
```

Comment:
Why you hate "capaz" so much?

Reply:
Às vezes com sinônimos soa melhor, mas eu posso reverter

Reply:
"entendê-la" me parece soar melhor aqui.

Reply:
Alterei apenas essa parte.